modules/microshift-embed-ostree-enable-eus-repos.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * microshift_install/microshift-embed-into-rpm-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enable-eus-repos_{context}"]
+= Enabling extended support repositories for image building
+
+If you have an extended support (EUS) release of {microshift-short} or {op-system-base-full}, you must enable the {op-system-base} EUS repositories for image builder to use. If you do not have an EUS version, you can skip these steps.
+
+.Prerequisites
+
+* You have an EUS version of {microshift-short} or {op-system-base}, or you are updating to one.
+* You have root-user access to your build host.
+* You reviewed the link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/installing/microshift-install-rpm#install-rpm-rhde-compatibility-table[{op-system-bundle} release compatibility matrix].
```

Comment:
I also mentioned this in my comment for the snippet, but the link here is taking me to the top of the page/chapter where the compatibility matrix is located, instead of taking me directly to the matrix.